src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react'
+import NavBar from './NavBar/NavBar'
+
+export default class Header extends Component {
```

Comment:
Why the "extends" is used here ?

Reply:
i just created it using rcc function akka i will change it